modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -2044,13 +2044,21 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/{page-ver
 a|
 Place a database into quarantine or remove it from it.
 | Signature
-m|dbms.quarantineDatabase(databaseName :: STRING, setStatus :: BOOLEAN, reason = No reason given :: STRING) :: (databaseName :: STRING, quarantined :: BOOLEAN, result :: STRING)
+m|dbms.quarantineDatabase(databaseName :: STRING, setStatus :: BOOLEAN, reason = No reason given :: STRING, wipeMode = DO_NOT_WIPE :: STRING) :: (databaseName :: STRING, quarantined :: BOOLEAN, result :: STRING)
 | Mode
 m|DBMS
 // | Default roles
 // m|admin
 |===
 
+[NOTE]
+====
+The `wipeMode` option is only enforced during the removal of a database from quarantine.
+The possible values for `wipeMode` are: `DO_NOT_WIPE` (default value and previous behaviour), `WIPE_STATE_KEEP_STORE` and `WIPE_STATE_AND_STORE`.
+Please keep in mind that the deletion of the cluster state and store (`WIPE_STATE_AND_STORE`) or even just the deletion of the state (`WIPE_STATE_KEEP_STORE`) could cause loss of data and should be carefully evaluated depending on the case.
```

Comment:
Maybe it's worth explaining that wiping the cluster state means this server will join the cluster as a new member. And in order for the cluster to accept new members, they need to have a quorum of old members. And therefore it's dangerous to wipe the state for >=quorum of members. Sometimes the quarantine might be due to the store state, so deleting the store only might be sufficient. But I see now that we don't have an option for it, let's discuss it.